node_virsh.sh

```diff
@@ -6,7 +6,8 @@ DIRECTIVE=${1-}
 
 set -u
 
-NODESXML=`ls node*.xml`
+CURRENT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
The (cd && pwd) serves to purposes: It produces the path from root to the script. If we only use BASH_SOURCE[0] we get the path from user to the script. It also changes the working directory to where the node xml files are for the scope of the script, this lets us call "$VIRSH define $XML" without adding a path to the $XML variable.